TestBattery.h

```diff
@@ -0,0 +1,25 @@
+#include "NewBatteryState.h"
+#include <assert.h>
+
+void testBattery()
+{
+    NewBatteryState n;
+    // temperature test
+    assert(n.checkBatteryIsOk(0, 30, 0.5) == true);
```

Comment:
Good coverage of tests for different values, well done !

Reply:
Thanks